fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsAccountTransactionRepository.java

```diff
@@ -49,4 +50,8 @@ List<SavingsAccountTransaction> findTransactionRunningBalanceBeforePivotDate(@Pa
 
     @Query("select sat from SavingsAccountTransaction sat where sat.refNo = :refNo")
```

Comment:
@rrpawar96 ... why is that additional @Query annotation needed here? Should just work the same way if you rename the function to "findByRefNo"... and is shorter... and you just have to "maintain" one thing (the function name). For short queries (i. e. not too many parameters) I think using the repository function magic is nicer.